robot/description/urdf/caffeine.gazebo.xacro

```diff
@@ -93,5 +93,31 @@
             <!-- TODO: add Gaussian noise based off Garmin GPS -->
         </plugin>
     </gazebo>
+
+    <!-- IMU -->
+    <gazebo reference="${tf_prefix}/imu_link">
+        <material>Gazebo/Black</material>
+
+        <gravity>true</gravity>
+        <sensor name="imu_sensor" type="imu">
+            <always_on>true</always_on>
+            <update_rate>128.0</update_rate>
+            <topic>__default_topic__</topic>
+            <!-- TODO: set Drift based off Phidget IMU -->
+
+            <plugin filename="libgazebo_ros_imu_sensor.so" name="imu_plugin">
```

Comment:
I think it's better if these go under constants

Reply:
which? The topic/body/frame names?